modules/aws-outposts-requirements-limitations.adoc

```diff
@@ -30,4 +36,8 @@ The Outpost has edge compute nodes, while the VPC has cloud-based compute nodes.
 The cloud-based AWS Elastic Block volume cannot attach to Outpost edge compute nodes, and the Outpost volumes cannot attach to cloud-based compute nodes.
 +
 As a result, you cannot use CSI snapshots to migrate applications that use persistent storage from cloud-based compute nodes to edge compute nodes or directly use the original persistent volume.
-To migrate persistent storage data for applications, you must perform a manual backup and restore operation.
\ No newline at end of file
+To migrate persistent storage data for applications, you must perform a manual backup and restore operation.
+
+* AWS Outposts does not support AWS Network Load Balancers or AWS Classic Load Balancers.
+To manages Ingress objects for your edge compute resources, you must install the AWS Load Balancer Operator so that you can use AWS Application Load Balancers in the AWS Outposts environment.
+If your cluster contains both edge and cloud-based compute instances that share workloads, additional configuration is required.
```

Comment:
No link to `Using the AWS Load Balancer Operator in an AWS VPC cluster extended into an Outpost` chapter?

Reply:
Unfortunately, the RH customer portal does not support links from anything in the `modules` directory :( Our usual workaround is to include an "Additional resources" list immediately after the module. I can add one with a link to the relevant content.